modules/cli/pages/index.adoc

```diff
@@ -171,11 +171,26 @@ where `<link-name>` is the name of the link specified during creation.
 [id="link-cost"] 
 == Specifying link cost
 
-When linking sites, you can assign a cost to each link to influence the traffic flow. By default, link cost is set to `1` for a new link.
-In a {service-network}, the routing algorithm attempts to use the path with the lowest total cost from caller to target service.
+When linking sites, you can assign a cost to each link to influence the traffic flow. 
+By default, link cost is set to `1` for a new link.
+In a {service-network}, the routing algorithm attempts to use the path with the lowest total cost from client to target server.
+
+* If you have services distributed across different sites, you might want a client to favor a particular target or link.
+In this case, you can specify a cost of greater than `1` on the alternative links to reduce the usage of those links.
++
+NOTE: The distribution of open connections is statistical, that is, not a round robin system.
+
+* If a connection only traverses one link, then the path cost is equal to the link cost.  
+If the connection traverses more than one link, the path cost is the sum of all the links involved in the path.  
+
+* Cost acts as a threshold for using a path from client to server in the network.
+When there is only one path, traffic flows on that path regardless of cost.  
++
+NOTE: If you start with two targets for a service, and one of the targets is no longer available, traffic flows on the remaining path regardless of cost.
+
+* When there are a number of paths from a client to a server, traffic flows on the lowest cost path until the number of connections exceeds the cost of an alternative path.
```

Comment:
"When there are a number of paths from a client to instances or a service,"

The current wording suggests that there are multiple paths to the same server and we're choosing which to use.  In reality, we are choosing between multiple instances of a service.